genie-web/src/main/java/com/netflix/genie/web/tasks/job/JobMonitor.java

```diff
@@ -205,14 +218,22 @@ public void run() {
      */
     @Override
     public GenieTaskScheduleType getScheduleType() {
-        return GenieTaskScheduleType.FIXED_DELAY;
+        return GenieTaskScheduleType.ADAPTIVE_BACKOFF;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public AdaptiveBackOff getGetAdaptiveBackoff() {
+        return adaptiveBackOff;
     }
 
     /**
      * {@inheritDoc}
      */
     @Override
-    public long getFixedDelay() {
-        return this.execution.getCheckDelay().orElseThrow(IllegalArgumentException::new);
+    public long getGetAdaptiveBackoffCheckInterval() {
```

Comment:
The method name looks odd - has 2 gets .

Reply:
ugh, thanks